app/pages/lab-pages-editor/helpers/createStep.js

```diff
@@ -0,0 +1,7 @@
+/*
+Creates a Step, with tasks, that's ready to be inserted into a Workflow.
+ */
+
+export default function createStep(stepKey, taskKeys = []) {
+  return [stepKey, { taskKeys }];
```

Comment:
```suggestion
  return [stepKey, { stepKey, taskKeys }];
```
You've missed the `stepKey` and `next` properties for the new step. The properties of each Step object should be listed in the Step model in the classifier code.

Reply:
Thanks Jim - I missed the second `stepKey` since I was referring to an old [ADR](https://github.com/zooniverse/front-end-monorepo/blob/master/docs/arch/adr-10.md) instead of the updated FEM spec files.